charts/pihole/0.0.1/questions.yaml

```diff
@@ -0,0 +1,214 @@
+    path: "/web"
+
+questions:
+  # Image related
+  - variable: image
+    description: "Docker Image Details"
+    label: "Docker Image"
+    group: "Container Images"
+    schema:
+      type: dict
+      required: true
+      attrs:
+        - variable: repository
+          description: "Docker image repository"
+          label: "Image repository"
+          schema:
+            type: string
+            required: true
+            default: "linuxserver/jackett"
+        - variable: tag
+          description: "Tag to use for specified image"
+          label: "Image Tag"
+          schema:
+            type: string
+            default: "version-v0.17.153"
+        - variable: pullPolicy
+          description: "Docker Image Pull Policy"
+          label: "Image Pull Policy"
+          schema:
+            type: string
+            default: "IfNotPresent"
+            enum:
+              - value: "IfNotPresent"
+                description: "Only pull image if not present on host"
+              - value: "Always"
+                description: "Always pull image even if present on host"
+              - value: "Never"
+                description: "Never pull image even if it's not present on host"
+
+  - variable: claimToken
+    label: "jackett Claim Token"
+    group: "jackett Configuration"
+    schema:
+      type: string
+      default: ""
+
+  - variable: extraEnv
+    label: "Environment variables for jackett"
+    group: "jackett Configuration"
+    schema:
+      type: dict
+      additional_attrs: true
+      attrs: []
+
+  - variable: proxy
+    label: "Configure Proxy environment variable"
+    group: "jackett Configuration"
+    schema:
+      type: dict
+      attrs:
+        - variable: enabled
+          label: "Enable Proxy"
+          schema:
+            type: boolean
+            default: false
+
+  - variable: hostNetwork
+    label: "Configure Host Network"
+    group: "Networking"
+    schema:
+      type: boolean
+      default: false
+
+  - variable: timezone
+    label: "jackett container timezone"
+    group: "jackett Configuration"
+    schema:
+      type: string
+      default: "Etc/UTC"
+      $ref:
+        - "definitions/timezone"
+
+  # Update strategy
+  - variable: strategyType
+    description: "Upgrade Policy"
+    label: "Update Strategy"
+    group: "Scaling/Upgrade Policy"
+    schema:
+      type: string
+      default: "Recreate"
+      enum:
+        - value: "RollingUpdate"
+          description: "Create new pods and then kill old ones"
+        - value: "Recreate"
+          description: "Kill existing pods before creating new ones"
+
+  # Port configuration
+  - variable: jackettService
+    label: "Configure jackett Service"
+    group: "Networking"
+    schema:
+      type: dict
+      attrs:
+        - variable: port
+          label: "Port to expose for jackett UI"
+          schema:
+            type: int
+            min: 2000
+            max: 65535
+            default: 32204
```

Comment:
Looks like there's lots of ports for this one. Not sure how you want it configured